blocks/editable/format-toolbar/index.js

```diff
@@ -131,20 +136,28 @@ class FormatToolbar extends Component {
 		}
 	}
 
+	isFormatActive( format ) {
+		return this.props.formats[ format ] && this.props.formats[ format ].isActive;
+	}
+
 	render() {
-		const { formats, focusPosition, enabledControls = DEFAULT_CONTROLS } = this.props;
+		const { formats, focusPosition, enabledControls = DEFAULT_CONTROLS, customControls = [] } = this.props;
 		const { isAddingLink, isEditingLink, newLinkValue, settingsVisible, opensInNewWindow } = this.state;
 		const linkStyle = focusPosition
 			? { position: 'absolute', ...focusPosition }
 			: null;
 
 		const toolbarControls = FORMATTING_CONTROLS
 			.filter( control => enabledControls.indexOf( control.format ) !== -1 )
-			.map( ( control ) => ( {
-				...control,
-				onClick: this.toggleFormat( control.format ),
-				isActive: !! formats[ control.format ],
-			} ) );
+			.concat( customControls )
```

Comment:
Should we concat before filtering? Thinking we could define customControls but decide to disable temporarily for some reason.

Reply:
I've changed this to only pass formatters that have been filtered through formattingControls on editable.